src/structures/Utils.ts

```diff
@@ -49,15 +55,16 @@ export abstract class TrackUtils {
 
     if (Array.isArray(trackOrTracks) && trackOrTracks.length) {
       for (const track of trackOrTracks) {
-        if (!(track[TRACK_SYMBOL] || track[UNRESOLVED_TRACK_SYMBOL])) return false
+        if (!(track[TRACK_SYMBOL] || track[UNRESOLVED_TRACK_SYMBOL]))
```

Comment:
Codacy has a fix for the issue: [Expected { after 'if' condition.](https://app.codacy.com/gh/MenuDocs/erela.js/pullRequest?prid=7585532)
```suggestion
        if (!(track[TRACK_SYMBOL] || track[UNRESOLVED_TRACK_SYMBOL]))
```